scripts/config_gen.php

```diff
@@ -235,8 +237,7 @@ function combine_includes($js, $js_compress, $css, $css_compress, $settings) {
     $js_hash = '';
     $css_hash = '';
     if ($css) {
-        $css_out = file_get_contents(VENDOR_PATH . "twbs/bootstrap/dist/css/bootstrap.min.css");
-        $css_out .= file_get_contents(VENDOR_PATH . "twbs/bootstrap-icons/font/bootstrap-icons.css");
+        $css_out = file_get_contents(VENDOR_PATH . "twbs/bootstrap-icons/font/bootstrap-icons.css");
```

Comment:
Please make sure this file is included after the bootstrap main css file.

Reply:
Actually, why don't we add the bootstrap-icons as well to the bootswatch pipeline?

Reply:
Great, thanks!